package.json

```diff
@@ -244,6 +250,7 @@
     "@azure/ms-rest-nodeauth": "3.0.5",
     "adal-node": "0.1.28",
     "form-data": "2.3.3",
+    "fs-extra": "^10.0.0",
```

Comment:
This should be a devDependency

Also make sure the major versions match between this and "@types/fs-extra" - you might have to downgrade this one which is fine